criteria/document-maturity.md

```diff
@@ -11,13 +11,15 @@ redirect_from:
 ## Requirements
 
 * A codebase MUST be versioned.
-* A codebase that is ready to use MUST only depend on other codebases that are also ready to use.
-* A codebase that is not yet ready to use MUST have one of the labels: prototype, alpha, beta or pre-release version.
+* A codebase version that is ready to use MUST only depend on other codebase versions that are also ready to use.
```

Comment:
Unless I misunderstand, if the point is to mention external dependencies (as opposed to self-dependencies), then this would be clearer to me in plural.

```suggestion
* A codebase version that is ready to use MUST only depend on other codebases versions that are also ready to use.
```